src/components/RenderRouter/ListItem.tsx

```diff
@@ -605,7 +605,7 @@ class ListItem extends React.Component<Props, State> {
       const item = items as OverseerData;
       const image = {
         src: `/static/photos/overseers/${item.FirstName}_${item.LastName}_app.jpg`,
-        alt: `${item.FirstName} ${item.LastName}`,
+        alt: `Official photo of ${item.FirstName} ${item.LastName}`,
```

Comment:
Using words like "photo" or "image" in alt text is not recommended as screen readers will announce that the element is an image. All this achieves is redundancy.

I would recommend `Head shot of ${item.FirstName} ${item.LastName}` as it adds useful context to the image.

Reply:
Thank you for the information on that. The change has been added.